plugins/main/bridge/bridge.go

```diff
@@ -378,7 +420,18 @@ func cmdDel(args *skel.CmdArgs) error {
 		return err
 	}
 
-	if err := ipam.ExecDel(n.IPAM.Type, args.StdinData); err != nil {
+	// Determine the plugin type
+	// TODO: Add logic to support dual-stack config with different
+	// plugin types for IPv4 vs. IPv6.
+	var plugin_type string
+	switch {
+	case n.IPAM.Type != "":
+		plugin_type = n.IPAM.Type
+	default:
+		plugin_type = n.IPAM6.Type
+	}
```

Comment:
Same response. I can either add this handling into this current pull request, or add it as a followup. My preference would be to do staged pull requests.